src/modules/terminalfont.c

```diff
@@ -43,6 +43,71 @@ static void printTerminalFontFromConfigFile(FFinstance* instance, const char* co
     printTerminalFont(instance, font);
 }
 
+static void printXCFETerminal(FFinstance* instance)
+{
+    FFstrbuf fontName;
```

Comment:
With the following suggestions in mind, i'm not sure if we need a strbuf. Possible i'm wrong at this one.

Reply:
I was just trying to avoid including strings.h altogether.

Reply:
I am too, but if we never need to modify the buffer we need neither strbufs nor strings.h

Reply:
I see. Will try. :)